distribution/pull_v2_test.go

```diff
@@ -205,3 +215,190 @@ func TestFormatPlatform(t *testing.T) {
 		}
 	}
 }
+
+func Test_v2Puller_pullSchema2Config_Basic(t *testing.T) {
```

Comment:
I think we generally don't use underscores in the test-names (no direct suggestions for a good name, but perhaps you could look around a bit for tests in this area; sticking with similar names can help to run a subset of tests (all tests with a common prefix, e.g.)